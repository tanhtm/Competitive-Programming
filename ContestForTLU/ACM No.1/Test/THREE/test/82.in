557985999000000000
