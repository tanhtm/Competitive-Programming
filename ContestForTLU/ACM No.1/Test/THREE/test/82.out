4 6 9 12 13 15 17 19 20 24 26 35 
1 5 7 10 14 16 21 22 23 25 28 30 32 33 36 37 
